scripts/o3de/o3de/disable_gem.py

```diff
@@ -88,12 +88,13 @@ def disable_gem_in_project(gem_name: str = None,
         return 1
 
     # remove the gem
-    error_code = cmake.remove_gem_dependency(enabled_gem_file, gem_json_data['gem_name'])
+    gem_name = gem_name if gem_name else gem_json_data.get('gem_name','')
```

Comment:
you can also do this
```suggestion
    gem_name = gem_name or gem_json_data.get('gem_name','')
```